Makefile

```diff
@@ -281,6 +282,13 @@ $(ISO_ROOT)/images/$(NGINX_IMAGE_NAME)-$(NGINX_IMAGE_VERSION).tar.gz: \
 	IMAGE=$(NGINX_IMAGE)
 $(ISO_ROOT)/images/$(NGINX_IMAGE_NAME)-$(NGINX_IMAGE_VERSION).tar.gz: \
 	IMAGE_TAG=$(NGINX_IMAGE_NAME):$(NGINX_IMAGE_VERSION)
+$(ISO_ROOT)/images/$(SALT_MASTER_IMAGE_NAME)-$(SALT_MASTER_IMAGE_VERSION).tar.gz:
+	mkdir -p $(dir $@)
+	docker build -t "$(SALT_MASTER_IMAGE_NAME):$(SALT_MASTER_IMAGE_VERSION)" images/salt-master
+	$(eval IMAGE=$(shell docker image inspect $(SALT_MASTER_IMAGE_NAME) --format {{.Id}}) )
+	$(eval IMAGE_TAG=$(SALT_MASTER_IMAGE_NAME):$(SALT_MASTER_IMAGE_VERSION) )
+	docker tag $(IMAGE) $(IMAGE_TAG)
```

Comment:
Is there a need to do it in 2 steps ? setting the tag during docker build is not enough ?

Reply:
I started doing it on two steps like the provided images but yes makes more sense to jump the tagging phase